### ISSUE TITLE
Windows decoration settings don't have the tab for window button configuration

### ISSUE THREAD

**Issue by eimi418** [KaOSx/apps#185]
#### Submission type

After the installation of KaOS 2019 (KDE Plasma 5.16.2) user cannot easily change the windows button configuration (for example, move close button to the left, as in Amiga/macOS style). 

In other KDE-specific distros (like KDE Neon) in System Settings/Appearance/Application Style/Windows Decorations, there is an additional tab with settings of buttons. KaOS doesn't have it, but under the "Windows Decoration" title shows the blurred edge of the windows style selector.

#### Expected behaviour

There should be a feature to set the windows button position and sequence somewhere in the settings,

#### Actual behaviour

In KaOS you need to manually edit KDE settings to configure button position.

#### Steps to reproduce the problem

Install KaOS 20190627
Try to configure Windows Decorations setting

**Comment by demmm** [comment]
KaOS does not change KWin or Plasma-Workspace, packaged as shipped upstream.  Is this not an addition that kde-neon adds(script/extra package)?

**Comment by eimi418** [comment]
Since KDE 3 it was always possible to quickly configure the buttons on
window bar – so when yesterday I've installed KaOS first time, I found it
very strange I can't find these options. KDE neon has it by default, the
same is in Kubuntu, OpenSUSE and Manjaro KDE.

**Comment by demmm** [comment]
OK, if all distros have it then there must be a setting that needs switching while building for KaOS, not clear what though.

**Comment by demmm** [comment]
Build logs of kwindowsystems, kdecoration or kwin show no signs of anything missing, see the output of kwin as example:
```
==> Extracting sources...
  -> Extracting kwin-5.16.5.tar.xz with bsdtar
==> Starting prepare()...
==> Starting build()...
-- The C compiler identification is GNU 8.3.0
-- The CXX compiler identification is GNU 8.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Installing in the same prefix as Qt, adopting their path scheme.
-- Looking for __GLIBC__
-- Looking for __GLIBC__ - found
-- Performing Test _OFFT_IS_64BIT
-- Performing Test _OFFT_IS_64BIT - Success
-- Performing Test HAVE_DATE_TIME
-- Performing Test HAVE_DATE_TIME - Success
-- Found KF5Config: /usr/lib/cmake/KF5Config/KF5ConfigConfig.cmake (found version "5.61.0") 
-- Found KF5ConfigWidgets: /usr/lib/cmake/KF5ConfigWidgets/KF5ConfigWidgetsConfig.cmake (found version "5.61.0") 
-- Found KF5CoreAddons: /usr/lib/cmake/KF5CoreAddons/KF5CoreAddonsConfig.cmake (found version "5.61.0") 
-- Found KF5Crash: /usr/lib/cmake/KF5Crash/KF5CrashConfig.cmake (found version "5.61.0") 
-- Found KF5GlobalAccel: /usr/lib/cmake/KF5GlobalAccel/KF5GlobalAccelConfig.cmake (found version "5.61.0") 
-- Found Gettext: /usr/bin/msgmerge (found version "0.20.1") 
-- Found KF5I18n: /usr/lib/cmake/KF5I18n/KF5I18nConfig.cmake (found version "5.61.0") 
-- Found KF5Init: /usr/lib/cmake/KF5Init/KF5InitConfig.cmake (found version "5.61.0") 
-- Found KF5Notifications: /usr/lib/cmake/KF5Notifications/KF5NotificationsConfig.cmake (found version "5.61.0") 
-- Found KF5Package: /usr/lib/cmake/KF5Package/KF5PackageConfig.cmake (found version "5.61.0") 
-- Found KF5Plasma: /usr/lib/cmake/KF5Plasma/KF5PlasmaConfig.cmake (found version "5.61.0") 
-- Found KF5WidgetsAddons: /usr/lib/cmake/KF5WidgetsAddons/KF5WidgetsAddonsConfig.cmake (found version "5.61.0") 
-- Found KF5WindowSystem: /usr/lib/cmake/KF5WindowSystem/KF5WindowSystemConfig.cmake (found version "5.61.0") 
-- Found KF5IconThemes: /usr/lib/cmake/KF5IconThemes/KF5IconThemesConfig.cmake (found version "5.61.0") 
-- Found KF5IdleTime: /usr/lib/cmake/KF5IdleTime/KF5IdleTimeConfig.cmake (found version "5.61.0") 
-- Found KF5Wayland: /usr/lib/cmake/KF5Wayland/KF5WaylandConfig.cmake (found version "5.61.0") 
-- Found KF5: success (found suitable version "5.61.0", minimum required is "5.58.0") found components:  Config ConfigWidgets CoreAddons Crash GlobalAccel I18n Init Notifications Package Plasma WidgetsAddons WindowSystem IconThemes IdleTime Wayland 
-- Found KF5Completion: /usr/lib/cmake/KF5Completion/KF5CompletionConfig.cmake (found version "5.61.0") 
-- Found KF5Declarative: /usr/lib/cmake/KF5Declarative/KF5DeclarativeConfig.cmake (found version "5.61.0") 
-- Found KF5KCMUtils: /usr/lib/cmake/KF5KCMUtils/KF5KCMUtilsConfig.cmake (found version "5.61.0") 
-- Found KF5KIO: /usr/lib/cmake/KF5KIO/KF5KIOConfig.cmake (found version "5.61.0") 
-- Found KF5TextWidgets: /usr/lib/cmake/KF5TextWidgets/KF5TextWidgetsConfig.cmake (found version "5.61.0") 
-- Found KF5NewStuff: /usr/lib/cmake/KF5NewStuff/KF5NewStuffConfig.cmake (found version "5.61.0") 
-- Found KF5Service: /usr/lib/cmake/KF5Service/KF5ServiceConfig.cmake (found version "5.61.0") 
-- Found KF5XmlGui: /usr/lib/cmake/KF5XmlGui/KF5XmlGuiConfig.cmake (found version "5.61.0") 
-- Found KF5: success (found suitable version "5.61.0", minimum required is "5.58.0") found components:  Completion Declarative KCMUtils KIO TextWidgets NewStuff Service XmlGui 
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.2") 
-- Performing Test HAVE_EGL
-- Performing Test HAVE_EGL - Success
-- Found EGL: /usr/lib/libEGL.so (found version "1.5") 
-- Found epoxy: /usr/lib/libepoxy.so  
-- Wayland: Egl requires Client
-- Found Wayland_Client: /usr/lib/libwayland-client.so (found version "1.17.0") 
-- Found Wayland_Cursor: /usr/lib/libwayland-cursor.so (found version "1.17.0") 
-- Found Wayland_Egl: /usr/lib/libwayland-egl.so (found version "18.1.0") 
-- Found Wayland: /usr/lib/libwayland-client.so;/usr/lib/libwayland-cursor.so;/usr/lib/libwayland-egl.so (found suitable version "1.17.0", minimum required is "1.2") found components:  Cursor Egl 
-- Found XKB: /usr/lib/libxkbcommon.so (found suitable version "0.8.4", minimum required is "0.7.0") 
-- Found Libinput: /usr/lib/libinput.so (found suitable version "1.14.1", minimum required is "1.9") 
-- Found UDev: /usr/include  
-- Found Libdrm: /usr/lib/libdrm.so (found suitable version "2.4.99", minimum required is "2.4.62") 
-- Found gbm: /usr/lib/libgbm.so (found version "19.1.6") 
-- Could NOT find libhardware (missing: libhardware_LIBRARY libhardware_INCLUDE_DIR) 
-- Could NOT find libhwcomposer (missing: libhwcomposer_LIBRARY libhwcomposer_INCLUDE_DIR) 
-- Could NOT find hybriseglplatform (missing: hybriseglplatform_LIBRARY hybriseglplatform_INCLUDE_DIR) 
-- Found X11: /usr/include   
-- Looking for XOpenDisplay in /usr/lib/libX11.so;/usr/lib/libXext.so
-- Looking for XOpenDisplay in /usr/lib/libX11.so;/usr/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found XCB_XCB: /usr/lib/libxcb.so (found version "1.13.1") 
-- Found XCB_RENDER: /usr/lib/libxcb-render.so (found version "1.13.1") 
-- Found XCB_SHAPE: /usr/lib/libxcb-shape.so (found version "1.13.1") 
-- Found XCB_XFIXES: /usr/lib/libxcb-xfixes.so (found version "1.13.1") 
-- Found XCB_SHM: /usr/lib/libxcb-shm.so (found version "1.13.1") 
-- Found XCB_COMPOSITE: /usr/lib/libxcb-composite.so (found version "1.13.1") 
-- Found XCB_CURSOR: /usr/lib/libxcb-cursor.so (found version "0.1.3") 
-- Found XCB_DAMAGE: /usr/lib/libxcb-damage.so (found version "1.13.1") 
-- Found XCB_GLX: /usr/lib/libxcb-glx.so (found version "1.13.1") 
-- Found XCB_ICCCM: /usr/lib/libxcb-icccm.so (found version "0.4.1") 
-- Found XCB_IMAGE: /usr/lib/libxcb-image.so (found version "0.4.0") 
-- Found XCB_KEYSYMS: /usr/lib/libxcb-keysyms.so (found version "0.4.0") 
-- Found XCB_RANDR: /usr/lib/libxcb-randr.so (found version "1.13.1") 
-- Found XCB_SYNC: /usr/lib/libxcb-sync.so (found version "1.13.1") 
-- Found XCB: /usr/lib/libxcb.so;/usr/lib/libxcb-render.so;/usr/lib/libxcb-shape.so;/usr/lib/libxcb-xfixes.so;/usr/lib/libxcb-shm.so;/usr/lib/libxcb-composite.so;/usr/lib/libxcb-cursor.so;/usr/lib/libxcb-damage.so;/usr/lib/libxcb-glx.so;/usr/lib/libxcb-icccm.so;/usr/lib/libxcb-image.so;/usr/lib/libxcb-keysyms.so;/usr/lib/libxcb-randr.so;/usr/lib/libxcb-sync.so (found suitable version "1.13.1", minimum required is "1.10") found components:  XCB XFIXES DAMAGE COMPOSITE SHAPE SYNC RENDER RANDR KEYSYMS IMAGE SHM GLX CURSOR ICCCM 
-- Found X11_XCB: /usr/lib/libX11-xcb.so (found version "1.6.8") 
-- Found Qt5FontDatabaseSupport: /usr/lib/libQt5FontDatabaseSupport.a (found version "5.13.1") 
-- Found Qt5ThemeSupport: /usr/lib/libQt5ThemeSupport.a (found version "5.13.1") 
-- Found Qt5EventDispatcherSupport: /usr/lib/libQt5EventDispatcherSupport.a (found version "5.13.1") 
-- Found Freetype: /usr/lib/libfreetype.so (found version "2.10.1") 
-- Found Fontconfig: /usr/lib/libfontconfig.so (found version "2.13.1") 
-- Found Xwayland: /usr/bin/Xwayland  
-- Found Libcap: /usr/sbin/setcap  
-- Found QtQuick-QMLModule: TRUE (found version "")  
-- Found QtQuick.Controls-QMLModule: TRUE (found version "")  
-- Found QtQuick.Layouts-QMLModule: TRUE (found version "")  
-- Found QtQuick.VirtualKeyboard-QMLModule: TRUE (found version "")  
-- Found QtQuick.Window-QMLModule: TRUE (found version "")  
-- Found QtMultimedia-QMLModule: TRUE (found version "")  
-- Found org.kde.kquickcontrolsaddons-QMLModule: TRUE (found version "")  
-- Found org.kde.plasma.core-QMLModule: TRUE (found version "")  
-- Found org.kde.plasma.components-QMLModule: TRUE (found version "")  
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for include file malloc.h
-- Looking for include file malloc.h - found
-- Looking for sys/prctl.h
-- Looking for sys/prctl.h - found
-- Looking for PR_SET_DUMPABLE
-- Looking for PR_SET_DUMPABLE - found
-- Looking for PR_SET_PDEATHSIG
-- Looking for PR_SET_PDEATHSIG - found
-- Looking for sys/procctl.h
-- Looking for sys/procctl.h - not found
-- Looking for PROC_TRACE_CTL
-- Looking for PROC_TRACE_CTL - not found
-- Looking for sys/sysmacros.h
-- Looking for sys/sysmacros.h - found
-- Looking for linux/vt.h
-- Looking for linux/vt.h - found
-- Looking for linux/fb.h
-- Looking for linux/fb.h - found
-- Looking for SCHED_RESET_ON_FORK
-- Looking for SCHED_RESET_ON_FORK - found
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
CMake Warning at /usr/lib/cmake/KF5Package/KF5PackageMacros.cmake:74 (message):
  KPackage components should be specified in reverse domain notation.
  Appstream information won't be generated for kcm_kwindecoration.
Call Stack (most recent call first):
  kcmkwin/kwindecoration/CMakeLists.txt:30 (kpackage_install_package)


CMake Warning at /usr/lib/cmake/KF5Package/KF5PackageMacros.cmake:74 (message):
  KPackage components should be specified in reverse domain notation.
  Appstream information won't be generated for kcm_kwin_virtualdesktops.
Call Stack (most recent call first):
  kcmkwin/kwindesktop/CMakeLists.txt:33 (kpackage_install_package)


CMake Warning at /usr/lib/cmake/KF5Package/KF5PackageMacros.cmake:74 (message):
  KPackage components should be specified in reverse domain notation.
  Appstream information won't be generated for kcm_kwin_effects.
Call Stack (most recent call first):
  kcmkwin/kwineffects/CMakeLists.txt:33 (kpackage_install_package)


-- The following features have been enabled:

 * Qt5Test (required version >= 5.12.0), Required for building tests
   Required for tests
 * KF5Activities (required version >= 5.58.0), Enable building of KWin with kactivities support
   Enable building of KWin with kactivities support
 * KF5DocTools (required version >= 5.58.0), Enable building documentation
   Enable building documentation
 * Breeze-Decoration, Default decoration plugin Breeze
 * Wayland::EGL, Enable building of Wayland backend and QPA with EGL support.
 * XInput, Required for poll-free mouse cursor updates
 * XCB-ICCCM, Required for building test applications for KWin
 * prctl/procctl tracing control, Required for disallowing ptrace on kwin_wayland process
 * linux/vt.h, Required for virtual terminal support under wayland
 * linux/fb.h, Required for the fbdev backend
 * SCHED_RESET_ON_FORK, Required for running kwin_wayland with real-time scheduling

-- The following RUNTIME packages have been found:

 * Qt5Multimedia
   Runtime-only dependency for effect video playback
 * KF5Kirigami2 (required version >= 5.58.0), A QtQuick based components set
   Required at runtime for Virtual desktop KCM and the virtual keyboard
 * EGL, A platform-agnostic mechanism for creating rendering surfaces for use with other graphics libraries, such as OpenGL|ES and OpenVG., <https://www.khronos.org/egl/>
   Required to build KWin with EGL support
 * Xwayland, Xwayland X server, <https://x.org>
   Needed for running kwin_wayland
 * QtQuick-QMLModule, QML module 'QtQuick' is a runtime dependency.
 * QtQuick.Controls-QMLModule, QML module 'QtQuick.Controls' is a runtime dependency.
 * QtQuick.Layouts-QMLModule, QML module 'QtQuick.Layouts' is a runtime dependency.
 * QtQuick.VirtualKeyboard-QMLModule, QML module 'QtQuick.VirtualKeyboard' is a runtime dependency.
 * QtQuick.Window-QMLModule, QML module 'QtQuick.Window' is a runtime dependency.
 * QtMultimedia-QMLModule, QML module 'QtMultimedia' is a runtime dependency.
 * org.kde.kquickcontrolsaddons-QMLModule, QML module 'org.kde.kquickcontrolsaddons' is a runtime dependency.
 * org.kde.plasma.components-QMLModule, QML module 'org.kde.plasma.components' is a runtime dependency.
 * org.kde.plasma.core-QMLModule, QML module 'org.kde.plasma.core' is a runtime dependency.

-- The following OPTIONAL packages have been found:

 * Qt5Test (required version >= 5.12.0), Required for building tests
   Required for tests
 * KF5ItemViews (required version >= 5.61.0)
 * KF5JobWidgets (required version >= 5.61.0)
 * KF5Solid (required version >= 5.61.0)
 * KF5Sonnet (required version >= 5.61.0)
 * KF5Attica (required version >= 5.61.0)
 * KF5NewStuffCore (required version >= 5.61.0)
 * KF5Auth (required version >= 5.61.0)
 * KF5Codecs (required version >= 5.61.0)
 * KF5Activities (required version >= 5.58.0), Enable building of KWin with kactivities support
   Enable building of KWin with kactivities support
 * KF5DocTools (required version >= 5.58.0), Enable building documentation
   Enable building documentation
 * Breeze (required version >= 5.9.0)
   For setting the default window decoration plugin
 * Libdrm (required version >= 2.4.62), Userspace interface to kernel DRM services., <https://wiki.freedesktop.org/dri/>
   Required for drm output on Wayland.
 * gbm, Mesa gbm library., <https://www.mesa3d.org>
   Required for egl output of drm backend.
 * X11_XCB, A compatibility library for code that translates Xlib API calls into XCB calls, <https://xorg.freedesktop.org/>
   Required for building X11 windowed backend of kwin_wayland
 * PkgConfig
 * Libcap, Capabilities are a measure to limit the omnipotence of the superuser., <https://sites.google.com/site/fullycapable/>
   Needed for running kwin_wayland with real-time scheduling policy

-- The following REQUIRED packages have been found:

 * Qt5Qml (required version >= 5.13.1)
 * Qt5QuickWidgets
 * Qt5Sensors
 * Qt5Script
 * Qt5UiTools
 * Qt5 (required version >= 5.12.0)
 * KF5Crash (required version >= 5.58.0)
 * KF5GlobalAccel (required version >= 5.58.0)
 * KF5Init (required version >= 5.58.0)
 * KF5Notifications (required version >= 5.58.0)
 * KF5Plasma (required version >= 5.58.0)
 * KF5WindowSystem (required version >= 5.58.0)
 * KF5IconThemes (required version >= 5.58.0)
 * KF5IdleTime (required version >= 5.58.0)
 * KF5Wayland (required version >= 5.58.0)
 * Qt5Quick (required version >= 5.11.0)
 * ECM (required version >= 1.6.0)
 * KF5Package (required version >= 5.61.0)
 * KF5Declarative (required version >= 5.58.0)
 * KF5KCMUtils (required version >= 5.58.0)
 * KF5Completion (required version >= 5.61.0)
 * Qt5Concurrent (required version >= 5.11.0)
 * KF5KIO (required version >= 5.58.0)
 * Gettext
 * KF5I18n (required version >= 5.61.0)
 * KF5TextWidgets (required version >= 5.58.0)
 * Qt5Network (required version >= 5.11.0)
 * KF5NewStuff (required version >= 5.58.0)
 * KF5Service (required version >= 5.58.0)
 * Qt5DBus (required version >= 5.11.0)
 * KF5CoreAddons (required version >= 5.61.0)
 * Qt5Widgets (required version >= 5.11.0)
 * KF5WidgetsAddons (required version >= 5.61.0)
 * KF5ConfigWidgets (required version >= 5.61.0)
 * KF5XmlGui (required version >= 5.58.0)
 * KF5 (required version >= 5.58.0)
 * Threads
   Needed for VirtualTerminal support in KWin Wayland
 * Qt5Gui (required version >= 5.12.0)
 * KDecoration2 (required version >= 5.13.0)
 * Qt5X11Extras
 * KScreenLocker
   For screenlocker integration in kwin_wayland
 * epoxy, libepoxy, <https://github.com/anholt/libepoxy>
   OpenGL dispatch library
 * Wayland (required version >= 1.2), C library implementation of the Wayland protocol: a protocol for a compositor to talk to its clients, <https://wayland.freedesktop.org/>
   Required for building KWin with Wayland support
 * XKB (required version >= 0.7.0), XKB API common to servers and clients., <https://xkbcommon.org>
   Required for building KWin with Wayland support
 * Libinput (required version >= 1.9), Library to handle input devices in Wayland compositors and to provide a generic X.Org input driver., <https://www.freedesktop.org/wiki/Software/libinput/>
   Required for input handling on Wayland.
 * UDev, Linux device library., <http://www.freedesktop.org/software/systemd/libudev/>
   Required for input handling on Wayland.
 * X11, X11 libraries, <https://www.x.org>
 * XCB (required version >= 1.10), X protocol C-language Binding, <https://xcb.freedesktop.org/>
 * Qt5FontDatabaseSupport, Qt FontDatabaseSupport module., <https://www.qt.io>
 * Qt5ThemeSupport, Qt ThemeSupport module., <https://www.qt.io>
 * Qt5EventDispatcherSupport, Qt EventDispatcherSupport module., <https://www.qt.io>
 * Freetype, A font rendering engine, <https://www.freetype.org>
   Needed for KWin's QPA plugin.
 * Fontconfig, Fontconfig is a library for configuring and customizing font access, <https://www.fontconfig.org/>
   Needed for KWin's QPA plugin.
 * Qt5Core

-- The following OPTIONAL packages have not been found:

 * libhybris, libhybris allows to run bionic-based HW adaptations in glibc systems., <https://github.com/libhybris/libhybris>
   Required for libhybris backend

-- Configuring done
-- Generating done
-- Build files have been written to: /buildsys/apps/kwin/src/build

```

**Comment by demmm** [comment]
Issue is in qtcurve, not KWin or packaging, just switch the application style to Breeze and the tabs show.

**Comment by eimi418** [comment]
Confirmed, quite unexpected. Qtcurve seems to have all settings inside,
kitchen sink included. Thanks.

**Comment by demmm** [comment]
This issue seems to be fixed in Plasma 5.17, closing.
Please reopen if you still see this issue.
